src/components/pagination/README.md

```diff
@@ -0,0 +1,29 @@
+# d2l-labs-pagination
+
+A component to indicate the existence of, and provide navigation for, multiple pages of content.
+
+## Usage
+
+```html
+<script type="module">
+    import '@brightspace-ui-labs/pagination/pagination.js';
+</script>
+<d2l-labs-pagination></d2l-labs-pagination>
+```
+
+## Properties
+
+| Property | Type | Description |
+|--|--|--|
+| `page-number` | Number (required), default: 1 | The current page number
```

Comment:
Yeah, I'd initially not included the `required`, but a teammate had suggested adding it since the initial documentation included it. Since I'm including the default value, I'll remove the `required` from there